frame/dvm/evm/precompiles/bridge/s2s/src/lib.rs

```diff
@@ -148,6 +156,23 @@ where
 		Ok(abi_encode_bytes(payload.encode().as_slice()))
 	}
 
+	fn create_opaque_call_payload(
```

Comment:
maybe, this name is not friendly to DApp developer. It will be exposed as a SDK API.
It returns the final cross-chain message payload, and it doesn't create anything but assemble incoming stuff, so I prefer `build_message_payload`.

Reply:
or just `build_message`

Reply:
`build_message` is too simple in this case. `assemble incoming stuff,` also means create in my view. So I prefer the current name style. Wait and see what others review opinions?

Reply:
see my solidity example below. the SDK will provide a pair methods: `sendMessage` and `buildMessage`. And `buildMessage` is just a wrapper for the precompile method. So, the similar name will be better.

Reply:
`create` means you create a new buisness thing.
`build` means the input and the output are the same business thing. only the format or encoding is different.